module.exports = async function handleException (exception) {
  console.error('[FABRIC:FUNCTIONS]', 'Exception:', exception);
}
